### PR TITLE
DFIR IRIS Integration contribution

### PR DESCRIPTION
**Pull request by Enigmatyk**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
New integration

## Description
IRIS is a collaborative platform aiming to help incident responders to share technical details during investigations

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py
Comment (yucohen) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:301]: if it just retrieves an str no need for outputs in this case.

**Review by yucohen — CHANGES_REQUESTED** [review]
Thanks for your contribution! Great job!
1. Please see my review comments.
2. make sure to fix all build steps.
3. Isn't mandatory, but adding a detailed description on how to get api tokens to `IRISDFIR_description.md` file would be really useful to the customer using this integration.

Dont hesitate to reach out if you have any questions here or on slack DFIR :)

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:138]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:160]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:175]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:190]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:301]: change the name to results to not create confusion.

**Comment by yucohen** [comment]
@Enigmatyk Great! approving!
waiting for your confirmation before merging :)

**Review by yucohen — APPROVED** [review]

**Review by yucohen — APPROVED** [review]
Great!!

**Pull request merged**
